build.sbt

```diff
@@ -61,7 +61,8 @@ lazy val microservice = (project in file("."))
 
 lazy val testSettings: Seq[Def.Setting[_]] = Seq(
   fork := true,
-  unmanagedSourceDirectories += baseDirectory.value / "test-utils"
+  unmanagedSourceDirectories += baseDirectory.value / "test-utils",
+  scalafmtOnCompile := true
```

Comment:
why are we doing this?

Reply:
When you run the tests it will run the scalafmt on it and tidy up the files, like it does with the code files. So hopefully avoiding the issues we have where we forget to run the scalafmt for the test files (which requires a separate command) and then the pipeline fails!